### ISSUE TITLE
stop telling us "let is the new var"

### ISSUE THREAD

**Issue by hayesmaker** [lukehoban/es6features#117]
let is not the new var.. they're different things, and var is still useful.

**Comment by zchrykng** [comment]
@hayesmaker perhaps you can enlighten us and provide examples. Where `var` is superiour to `const` or `let`.

**Comment by getify** [comment]
I agree with @hayesmaker that we should be using `var` alongside `let` (and `const`) in code. That's what I teach in all my classes and write about in my YDKJS books. Here's some other places I've addressed the topic:

* https://davidwalsh.name/for-and-against-let
* https://github.com/getify/Functional-Light-JS/blob/master/ch6.md#reassignment

**Comment by zchrykng** [comment]
Those are some good points. And I would never say that `var` should go away completely but rather it should only be used when you specifically need it and the `let` and `const` should be used whenever possible as they help prevent fairly common classes of problems. Maybe I am just overly used to block scoping since I have mainly been using C++ and Go recently.

**Comment by pielegacy** [comment]
@zchrykng my thoughts exactly, never liked the functionality of var so for me let was a welcome addition
